docs/ember-data/models.md

```diff
@@ -23,8 +23,8 @@ import Model, { attr } from '@ember-data/model';
 import CustomType from '../transforms/custom-transform';
 
 export default class User extends Model {
-  @attr()
-  name?:  string;
+  @attr('string')
+  declare name?:  string;
```

Comment:
Let's split this one and the one like it below into a separate PR: that way we can land the typo fixes while we think about/chat with Data folks about this part.

Reply:
Done.